meta-resin-morty/recipes-connectivity/dnsmasq/dnsmasq_%.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_remove = "file://0001-Treat-REFUSED-not-SERVFAIL-as-an-unsuccessful-upstre.patch"
```

Comment:
Will do, even though this means we don't treat meta-resin-common as previously said. This makes it not common anymore